taipy/core/taipy.py

```diff
@@ -71,6 +71,8 @@ def set(entity: Union[DataNode, Task, Sequence, Scenario, Cycle]):
         return _TaskManagerFactory._build_manager()._set(entity)
     if isinstance(entity, DataNode):
         return _DataManagerFactory._build_manager()._set(entity)
+    if isinstance(entity, Submission):
```

Comment:
ah yep let me add that. I will also make Submission's `is_editable` to return True as well since it now contains some user information. And we're adding set Submission to tp.set